src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,28 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        return (userService.findByEmail(email) != null && password.equals(
+                user.getPassword())) ? true : false;
```

Comment:
It already returns boolean, so this ternary operator is redundant
```suggestion
        return (userService.findByEmail(email) != null && password.equals(
                user.getPassword()));
```